src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -3905,3 +3905,6 @@ body.tribe-welcome #fs_connect {
 		width: 300px;
 	}
 }
+
+/* Upsell CSS */
+@import "upsell.pcss";
```

Comment:
I think that if we could put this inside a folder (can be `tribe-common-admin` or `tribe-admin`) as we do in other places it'd be perfect, so that we can start moving things there and keep it more organized. 

For example, how we do it for tickets commerce. Where we [have a main file](https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/postcss/tickets-commerce.pcss) (in this case this would be the main file) and it has calls to an [_all.pcss](https://github.com/the-events-calendar/event-tickets/blob/master/src/resources/postcss/tickets-commerce/_all.pcss) file inside of [the folder](https://github.com/the-events-calendar/event-tickets/tree/master/src/resources/postcss/tickets-commerce). And that `_all.pcss` has calls to the different sections. 

That way we can start organizing the admin files and grow from here. This file is already around 4K lines and I'd love to see us splitting this in the future. Having this new structure can be a great beginning.

Reply:
Gotcha. I notice that imports have to be done at the beginning of a pcss file, before any other statements, so I created a `tribe-common-admin` folder, then moved all the CSS from the `tribe-common-admin.pcss` file into `tribe-common-admin/_main.pcss` so I could import them in order within the `_all.pcss` file.